src/storage/mvcc/meta.rs

```diff
@@ -13,9 +13,14 @@
 
 use std::slice::Iter;
 use protobuf::core::Message;
+use protobuf::RepeatedField;
 use kvproto::mvccpb::{Meta as PbMeta, MetaItem, MetaLock};
 use super::Result;
 
+pub const META_SPLIT_SIZE: usize = 128;
```

Comment:
@ngaut Currently meta start splitting when it has 128 versions and keeps 32 latest versions in first meta. Do you suggest it to keep 20 latest versions instead of 32?